modules/Hoot/ui/managePanel/jobs.js

```diff
@@ -176,10 +176,13 @@ export default class Jobs extends Tab {
     async loadJobs() {
         let jobsRunning = await Hoot.api.getJobsRunning();
         let jobsHistory = await Hoot.api.getJobsHistory(this.params);
-        await Hoot.layers.refreshLayers();
+
+        if ( jobsHistory.total > 0 && this.total ) {
```

Comment:
Don't we want to compare the jobs total in the current response with the existing jobs total and only call refresh layer if the total increases?  I think this will still call refresh layers when the total is 10 but has not changed.